linkerd/app/core/src/transport/labels.rs

```diff
@@ -11,7 +11,11 @@ use std::fmt;
 /// Implements `FmtLabels`.
 #[derive(Clone, Debug, Eq, PartialEq, Hash)]
 pub enum Key {
-    Accept(Direction, TlsStatus),
+    Accept {
+        direction: Direction,
+        identity: TlsStatus,
+        local_addr: SocketAddr,
```

Comment:
How does this work on the outbound side? It's not a _local_ addr, is it? but really the _target_ addr? Perhaps we should just call it _target addr_ and encode the whole ip:port so that it makes sense on the outbound side?

Reply:
On the outbound side, it's set from the outbound listener's local address, _not_ the original destination (target) addr:
https://github.com/linkerd/linkerd2-proxy/blob/b0d95e619c3e1afffc4b8e6b19681a004ba70fea/linkerd/app/outbound/src/target.rs#L88

Is that what we want? If we're changing this to `target_addr`, it seems like it should be the original dest addr. In that case, we might want it to be the original dest addr in the inbound direction as well --- if a pod is listening on multiple ports, we would then capture which port the traffic was intended for by the client.

Reply:
Yeah, I think it's generally more useful to use the original dst / target addr. We're missing this data now (on the inbound side) and it's somewhat painful. In the case of the admin server, we'll have no original dst so we should just use the local target addr...